solutions/control-flow/product-sum.js

```diff
@@ -0,0 +1,30 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+    idx++;
+    return data[idx - 1].trim();
+}
+
+// -------- Do NOT edit anything above this line ----------
+// Use readLine() for taking input, it will read one line of from the input  and is stored in string format
+let number = parseInt(readLine());
+if(number==0)
```

Comment:
please follow coding conventions (spaces around operators) - [Coding Conventions](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#coding-conventions)

Reply:
Please add spaces around operators in below logic also